pocs/utils/rs232.py

```diff
@@ -141,7 +141,7 @@ def write(self, value):
         if self.is_threaded:
             response = self._serial_io.write(value)
         else:
-            response = self.ser.write(value.encode())
+            response = self.ser.write("{}\n".format(value).encode())
```

Comment:
Another comment on this same line: appending a \n is going to change the behavior enough that my tests will fail. Oh, and how is the mount going to respond to getting those \n values? And the dome?

I think you should drop the \n.

Reply:
Agreed, not the appropriate place for it and will be removing.

We should probably move toward the SerialIO concept as well as the start/end bytes as commented on below. I'm going to leave that to a separate PR and only make changes to the actual infineon board code here.